projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -1544,7 +1543,9 @@ public void watchJobs(
       runningJobExecutionsConsumer.accept(runningExecutionIds);
       resourceVersion = jobList.getMetadata().getResourceVersion();
     } catch (ApiException ex) {
-      log.info("Failed to list jobs for watching. Error was: {}", ex.getMessage());
+      log.info(
```

Comment:
what about 
```
log.info("Failed to list jobs for watching.", ex);
```

That looks cleaner to me.

Reply:
This I think would print a stack trace. And I want to avoid printing stack trace in info logs. As they indicate errornoues case. 
Imagine administrator looking at logs to see why is something not working and noticing bunch of stack traces that may mislead them .

Reply:
Oh sorry I missed the point.